src/prism.c

```diff
@@ -17238,7 +17236,7 @@ pm_serialize(pm_parser_t *parser, pm_node_t *node, pm_buffer_t *buffer) {
 PRISM_EXPORTED_FUNCTION void
 pm_serialize_parse(pm_buffer_t *buffer, const uint8_t *source, size_t size, const char *data) {
     pm_options_t options = { 0 };
-    if (data != NULL) pm_options_read(&options, data);
+    pm_options_read(&options, data);
```

Comment:
There are still 2 `if (data != NULL) pm_options_read(&options, data);` in `templates/src/serialize.c.erb`, those should be changed too to properly set the default start line.

Reply:
Alternatively, `pm_options_t options = { 0 };` is kind of always wrong (it sets start line to 0), so maybe that should be replaced by
```c
pm_options_t options;
pm_options_init(&options);
```
or so (I would declare that function as static inline in `include/prism/options.h` just below the struct).

Reply:
Thanks, I fixed the missing callsite. `pm_options_init` would work but is a bit finicky to get right.